CONFIG.md

```diff
@@ -291,6 +291,30 @@ the properties section in the JSON configuration file. Lastly, the client calls
 
 For information about the path resolution, please refer to the [Path Resolution](#path-resolution-and-object-loading) section.
 
+### Enabling Hazelcast Cloud Discovery
+The purpose of Hazelcast Cloud Discovery is to provide clients to use IP addresses provided by `hazelcast orchestrator`. To enable Hazelcast Cloud Discovery, specify a token for the `discoveryToken` field and set the `enabled` field to "true".
```

Comment:
I think what `a token` is is not clear here. Maybe you should express where this token comes from (It is provided by hazelcast orchestrator I assume)

Reply:
For now, I don't know how user gets the token. When Hazelcast Cloud is out, I think it becomes clear.